### PR TITLE
refactor: modernize sample test

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: samples/package.json [hunk 1]
```diff
@@ -9,13 +8,13 @@
     "node": ">=8"
```
Comment (schmidt-sebastian) [inline samples/package.json]: Should we update the engines field here as well?

**Comment by bcoe** [comment]
while you're at it, if you toss `nyc` in front of the `sample-test` stanza in package.json, hour coverage should go up.

**Comment by JustinBeckwith** [comment]
Any reason not to use c8 in this case?

**Comment by bcoe** [comment]
@JustinBeckwith no, I'd be excited to continue rolling it out; it will be slightly awkward as we phase out `.nycrc` for `.c8rc`; I was thinking maybe I should make `c8` load `.nycrc`.

**Pull request merged**